### PR TITLE
Fixed broken urls on docs build report

### PR DESCRIPTION
**Pull request by mikefrobbins**
## Description

Fixed broken links in reference documentation based on the docs repo build report

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/KeyVault/KeyVault/help/Undo-AzKeyVaultSecretRemoval.md [hunk 1]
```diff
@@ -162,6 +162,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Remove-AzKeyVaultSecret](./Remove-AzKeyVaultSecret.md)
 
-[Add-AzKeyVaultSecret](./Add-AzKeyVaultSecret.md)
```
Comment (BethanyZhou) [inline src/KeyVault/KeyVault/help/Undo-AzKeyVaultSecretRemoval.md:165]: Is this removed by mistake?

File: src/Monitor/Monitor/help/Get-AzActivityLogAlert.md [hunk 2]
```diff
@@ -128,8 +128,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Set-AzActivityLogAlert](./Set-AzActivityLogAlert.md)
 
-[Update-AzActivityLogAlert](./Update-AzActivityLogAlert.md)
```
Comment (BethanyZhou) [inline src/Monitor/Monitor/help/Get-AzActivityLogAlert.md:131]: Is this removed by mistake?

File: src/KeyVault/KeyVault/help/Undo-AzKeyVaultSecretRemoval.md (hunk 1 above)
Comment (mikefrobbins) [reply src/KeyVault/KeyVault/help/Undo-AzKeyVaultSecretRemoval.md:165]: The removal was intentional. The link is invalid due to no PowerShell command by that name existing in any of the Az PowerShell modules.

File: src/Monitor/Monitor/help/Get-AzActivityLogAlert.md (hunk 2 above)
Comment (mikefrobbins) [reply src/Monitor/Monitor/help/Get-AzActivityLogAlert.md:131]: The removal was intentional. The link is invalid due to no PowerShell command by that name existing in any of the Az PowerShell modules.

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**